### PR TITLE
Boilerplate setup

### PR DESCRIPTION
**Pull request by dginelsa**
- Set template to iOS mode
- Added Angular PWA Package: adds icon images and manifest.webmanifest file
- Added “.angular” to .gitignore to ignore Angular cache files
- Added Angular Async local storage library: https://github.com/cyrilletuzi/angular-async-local-storage

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
   <title>Ionic App</title>
 
-  <base href="/" />
+  <base href="/"/>
 
-  <meta name="color-scheme" content="light dark" />
-  <meta name="viewport" content="viewport-fit=cover, width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-  <meta name="format-detection" content="telephone=no" />
-  <meta name="msapplication-tap-highlight" content="no" />
+  <meta name="color-scheme" content="light dark"/>
```
Comment (kaisenl) [inline src/index.html:10]: It's fine, but did your editor automatically removing the trailing spaces on the tags?

File: package.json [hunk 2]
```diff
@@ -19,12 +19,14 @@
...
     "@angular/platform-browser-dynamic": "^15.0.0",
     "@angular/router": "^15.0.0",
+    "@angular/service-worker": "^15.0.0",
     "@capacitor/app": "4.1.1",
     "@capacitor/core": "4.7.0",
     "@capacitor/haptics": "4.1.0",
     "@capacitor/keyboard": "4.1.1",
     "@capacitor/status-bar": "4.1.1",
     "@ionic/angular": "^6.1.9",
+    "@ngx-pwa/local-storage": "^15.0.0",
```
Comment (kaisenl) [inline package.json:29]: Do we also want to add avataaar while we're here?

**Review by kaisenl — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (dginelsa) [reply package.json:29]: Let's hold off on that. I tested it and you can't customize the layout. Better to use the original JS library.

File: src/index.html (hunk 1 above)
Comment (dginelsa) [reply src/index.html:10]: I'll look out for that

File: package.json (hunk 2 above)
Comment (kaisenl) [reply package.json:29]: Cool, sounds good. Go ahead and merge it

**Pull request merged**